lib/active_merchant/billing/gateways/conekta.rb

```diff
@@ -173,23 +173,28 @@ def parse(body)
         JSON.parse(body)
       end
 
-      def headers(meta)
+      def headers(options)
         {
-          'Accept' => "application/vnd.conekta-v#{options[:version]}+json",
-          'Accept-Language' => 'es',
-          'Authorization' => 'Basic ' + Base64.encode64("#{options[:key]}:"),
-          'RaiseHtmlError' => 'false',
-          'Conekta-Client-User-Agent' => {'agent'=>"Conekta ActiveMerchantBindings/#{ActiveMerchant::VERSION}"}.to_json,
-          'X-Conekta-Client-User-Agent' => user_agent,
-          'X-Conekta-Client-User-Metadata' => meta.to_json
+          "Accept" => "application/vnd.conekta-v#{@options[:version]}+json",
```

Comment:
I thought `RuboCop` wanted exactly the opposite of these (Travis seems to agree).